packages/libs/eda/src/lib/map/analysis/mapTypes/shared.tsx

```diff
@@ -547,17 +549,36 @@ export const GLOBAL_VIEWPORT = {
  */
 
 const noDataPatterns = [
-  /did not contain any data/, // comes from map-markers endpoint
   /Could not generate continuous variable metadata/, // comes from continuous-variable-metadata endpoint
 ];
 
 export function isNoDataError(error: unknown) {
+  if (error instanceof Error && error.name === 'NoDataError') return true;
```

Comment:
```suggestion
  if (error instanceof NoDataError) return true;
```

This works for me (with the appropriate import). Did you try this?

Reply:
No, but I thought it would work. Instead, [I mocked a different custom error subclass we use](https://github.com/VEuPathDB/web-monorepo/blob/2512f4540573ecb57cfc00d9e14242a729db2f45/packages/libs/wdk-client/src/Service/ServiceError.ts#L13-L15).